CNAME

```diff
@@ -1 +0,0 @@
-railsgirlssummerofcode.org
```

Comment:
hey @geisasantos, this should *not* be removed from our CNAME file, only from your local fork (if it's causing you problems). Can you revert this commit?